torch/csrc/jit/passes/quantization.cpp

```diff
@@ -157,47 +219,38 @@ Node* insertObserver(
   return call;
 }
 
-c10::optional<QConfig> getQConfig(
-    const std::string& key,
-    const c10::optional<QConfig>& parent_qconfig,
-    const QConfigDict& qconfig_dict) {
-  if (qconfig_dict.find(key) != qconfig_dict.end()) {
-    return qconfig_dict.at(key);
-  }
-  return parent_qconfig;
-}
-
-void getQConfigMapHelper(
+void getQConfigMap(
```

Comment:
The name is now misleading, because `get` usually returns something.

Reply:
changing to fillQConfigMap